docs/earthfile/earthfile.md

```diff
@@ -584,6 +584,28 @@ For more information see the [multi-platform guide](../guides/multi-platform.md)
 
 Same as [`FROM --allow-privileged`](#allow-privileged).
 
+## VERSION
+
+#### Synopsis
+
+* `VERSION [--use-copy-include-patterns] <version-number>`
+
+#### Description
+
+The command `VERSION` identifies which set of features to enable in Earthly while handling the corresponding Earthfile. The `VERSION` command is optional; however if specified, it must be the first command in the Earthfile.
+
+| Version number | enabled features |
+| --- | --- | --- |
+| `0.5` | _initial functionality will be preserved_ |
+
+#### Options
+
+##### `--use-copy-include-patterns`
+
+Instructs Earthly to set buildkit's `include pattern` when executing `COPY` commands; without this feature, Earthly sends all files in the current build context except ignored files.
+Setting this option may increase the speed of COPY commands by forcing Earthly to only transfered files needed for the specific COPY, rather than transfering all files initially.
+This is disabled by default under `0.5`.
```

Comment:
I think we should have a separate page with a table of feature flags like this one. We should make it clear that all feature flags are experimental.

Reply:
Ok

Reply:
If we list them on a second page, would we then want to change 

```
VERSION [--use-copy-include-patterns] <version-number>
```

to something like

```
VERSION [<feature-flags>...] <version-number>
```

so we wouldn't have to manage this list across two different files?